src/index.js

```diff
@@ -6,12 +6,16 @@ import program from "commander";
 import pck from "./../package.json";
 
 program.version(pck.version)
-       .option('-u --username [username]', 'Your del.icio.us username')
```

Comment:
Although I changed the username to an argument in my branch so I don't think that's what's causing it...